sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -184,7 +184,7 @@ case class FileSourceScanExec(
     "Batched" -> supportsBatch.toString,
     "PartitionFilters" -> partitionFilters.mkString("[", ", ", "]"),
     "PushedFilters" -> dataFilters.mkString("[", ", ", "]"),
-    "InputPaths" -> relation.location.paths.mkString(", "))
+    "RootPaths" -> relation.location.rootPaths.mkString(", "))
```

Comment:
Btw, it would be nice to make sure the physical plan still has a good debug string when you call explain (i.e. tells which catalog it's using) since that will greatly impact performance in this case.